editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -4385,6 +4385,10 @@ void _insert_rid_recursive(Node *node, HashSet<RID> &rids) {
 		rids.insert(co->get_rid());
 	}
 
+	if (node->is_class("CSGShape3D")) {
```

Comment:
```suggestion
	} else if (node->is_class("CSGShape3D")) {
```
These conditions are mutually exclusive, so no reason to check both.

Reply:
Done